libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -140,6 +140,7 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM
     // return true if the estimate is valid
     bool get_relative_position_NED_home(Vector3f &vec) const override;
     bool get_relative_position_NED_origin(Vector3f &vec) const override;
+    void get_relative_position_NEU_origin_cm(Vector3f &vec) const;
```

Comment:
I actually think that NEU is easier to work with. So I'm not against adding methods to return values in that coordinate system, but if we do it, we should add those for all methods, not just one.

Regarding centimeters, I disagree with you - having one method be different than all others is more error prone. Besides making the API inconsistent - we need to improve our library APIs, not make them worse. Returning meters also goes in the direction of using SI base units where possible and converting it to another unit where it is needed.
I'll also mention that if some place needs meters, we would be doing `* 100 / 100`, which does more operations than needed.